vscode/src/azure/commands.ts

```diff
@@ -217,32 +202,6 @@ export async function initAzureWorkspaces(context: vscode.ExtensionContext) {
         await saveWorkspaceList();
         // Just kick off the refresh loop, no need to await
         startRefreshCycle(workspaceTreeProvider, workspace);
-
-        // Check if the storage account has CORS configured correctly.
```

Comment:
Now we use the Azure Quantum service proxy, not storage directly, no CORS is no longer an issue. All dead code.